dev/dvm/lib/dvm.rb

```diff
@@ -1,3 +1,7 @@
+module DVM
+  class DVMArgumentError < RuntimeError; end
+end
```

Comment:
@sdelano my best guess is that when I changed over to avoid having to package and install a dvm gem into the chef-server runtime, I somehow nuked that definition - because since then, we've been getting singularly unhelpful undefined class errors instead of something useful.